packages/terra-menu/tests/nightwatch/menu/BoundedMenu.jsx

```diff
@@ -13,6 +13,10 @@ class BoundedMenu extends React.Component {
     this.state = { open: true };
   }
 
+  componentDidMount() {
+    this.forceUpdate();
```

Comment:
What is forceUpdate used for in the componentDidMount lifecycles in nightwatch examples?

Reply:
That is the only way to to launch a popup on page load, as there will be no targetRef available before the content has been rendered once. So you have to force a second render after the component mounts. This previously worked in the tests, because tether would place the component on screen, even though it wouldn't position it, and the menu tests were only testing menu functionality. Hookshot will leave the component hidden if there is no targetRef.

Reply:
That could be updated in the menu PR following this to use a click to launch the menu, but an early pass at this made me realize it was too large in scope for this PR.

Reply:
Ok I'm cool with updating in the menu PR.